### PR TITLE
Add table for FreeRTOS scheduling algorithm configuration

### PR DESCRIPTION
**Pull request by kar-rahul-aws**
*Issue #, if available:*
https://forums.freertos.org/t/suggestions-for-mastering-the-freertos-real-time-kernel-1-0/19490/3

*Description of changes:*
This PR adds changes as requested in the forum for Chapter4.

1. Adds a table for FreeRTOS scheduling algorithm configuration
2. Fixes the output for **Example 4.6**

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: ch04.md [hunk 1]
```diff
@@ -2159,15 +2159,21 @@ scheduling algorithm does not guarantee time is shared equally between
 tasks of equal priority, only that *Ready* state tasks of equal priority
 enter the *Running* state in turn.
 
+<a name="tbl5" title="Table 5 The FreeRTOSConfig.h settings that configure the kernel to use various scheduling algorithms"></a>
```
Comment (aggarg) [inline ch04.md:2162]:
```suggestion
<a name="tbl5" title="Table 5 The FreeRTOSConfig.h settings to configure the kernel scheduling algorithm"></a>
```

File: ch04.md [hunk 2]
```diff
@@ -2159,15 +2159,21 @@ scheduling algorithm does not guarantee time is shared equally between
...
+
+***
 | Scheduling Algorithm            | Prioritized | `configUSE_PREEMPTION` | `configUSE_TIME_SLICING` |
 |---------------------------------|-------------|------------------------|--------------------------|
 | Preemptive With Time Slicing    | Yes         | 1 | 1   |
 | Preemptive Without Time Slicing | Yes         | 1 | 0   |
 | Co-Operative                    | No          | 0 | Any |
 
+***Table 5*** *The FreeRTOSConfig.h settings that configure the kernel to use various scheduling algorithms*
```
Comment (aggarg) [inline ch04.md:2171]: Same as above.

File: toc.md [hunk 3]
```diff
@@ -603,4 +603,5 @@
 + [Table 2 TickType\_t data type and the configTICK\_TYPE\_WIDTH\_IN\_BITS configuration](ch02.md#tbl2)
 + [Table 3 Macro prefixes](ch02.md#tbl3)
 + [Table 4 Common macro definitions](ch02.md#tbl4)
-+ [Table 5 The Effect of the uxBitsToWaitFor and xWaitForAllBits Parameters](ch09.md#tbl5)
++ [Table 5 The FreeRTOSConfig.h settings that configure the kernel to use various scheduling algorithms](ch04.md#tbl5)
```
Comment (aggarg) [inline toc.md:606]: Same as above.

File: ch04.md (hunk 1 above)
Comment (kar-rahul-aws) [reply ch04.md]: updated.

File: toc.md (hunk 3 above)
Comment (kar-rahul-aws) [reply toc.md]: updated.

File: ch04.md (hunk 2 above)
Comment (kar-rahul-aws) [reply ch04.md]: updated.

**Review by aggarg — APPROVED** [review]

**Pull request merged**